introduction.md

```diff
@@ -1,2 +1,5 @@
-I'm not going to tell you anything about myself. This is just a test file.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?